README.md

```diff
@@ -25,28 +25,29 @@ branch
 [r21.11](https://github.com/triton-inference-server/model_analyzer/tree/r21.11).**
 
 Triton Model Analyzer is a CLI tool to help with better understanding of the
-compute and memory requirements of the Triton Inference Server models. These
+compute and memory requirements of the
+[Triton Inference Server](https://github.com/triton-inference-server/server/) models. These
 reports will help the user better understand the trade-offs in different
 configurations and choose a configuration that maximizes the performance of
 Triton Inference Server.
 
-#### Model Analyzer can optimize for the following user constraints
-* latency
-* throughput
-* GPU memory
-
-#### Model Analyzer can automatically sweep the following variables and provide results across the resulting range:
-* [preferred batch size in triton server dynamic batching](https://github.com/triton-inference-server/server/blob/main/docs/optimization.md#dynamic-batcher)
-* [number of concurrent model instance loaded in triton](https://github.com/triton-inference-server/server/blob/main/docs/architecture.md#concurrent-model-execution)
-* [number of concurrent requests sent by the perf analyzer client](https://github.com/triton-inference-server/server/blob/main/docs/perf_analyzer.md#request-concurrency)
-
-#### Model Analyzer can manually sweep the following variables set by the user
-* framework
-* hardware (CPU or GPU)
-* model precision (for the TensorRT model only)
-* [queue delay in dynamic batching](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#delayed-batching)
-* [Rate Limiter Priority options](https://github.com/triton-inference-server/server/blob/main/docs/rate_limiter.md#rate-limiter) (eg. a model instance with priority 2 will be given half the number of scheduling changes)
-* [batch size of requests sent by the perf analyzer client](https://github.com/triton-inference-server/server/blob/main/docs/perf_analyzer.md#input-data)
+## Features
+
+* [Automatic and manual configuration search](docs/config_search.md). Model Analyzer can
+help you automatically find the optimal settings for
+[Dynamic Batching](docs/model_configuration.md#dynamic-batcher) and
```

Comment:
This link (as well as the model_configuration one on the next line) is broken. They need to point to triton server docs.

Reply:
Good catch. Fixed.